kubernetes-csi-node-driver-registrar-2.9.yaml

```diff
@@ -1,10 +1,14 @@
 package:
-  name: kubernetes-csi-node-driver-registrar
+  # Supported versions policy: https://kubernetes-csi.github.io/docs/node-driver-registrar.html
+  name: kubernetes-csi-node-driver-registrar-2.9
   version: 2.9.0
-  epoch: 0
+  epoch: 1
```

Comment:
I'm bumping this so that the `provides: ${{package.full-version}}` wins against the pre-existing packages.